Automattic/wp-calypso
# Media Library: Display File Size for Individual Items

#### Changes proposed in this Pull Request

* Allows users to see the size of each file they upload

#### Testing instructions

You'll need to apply Automattic/jetpack#18637 for this to work!

Visit the Media Library and open a file, then verify that the "File Size" is displayed neatly in the corner which accurately corresponds to the size stated in WP-Admin. 

<img width="1443" alt="Screenshot 2021-01-30 at 20 51 09" src="https://user-images.githubusercontent.com/43215253/106367803-3f3c3f80-633d-11eb-8b99-6a7ef70fa67f.png">

Fixes #49492

cc @sgomes
